app/models/katello/system.rb

```diff
@@ -63,6 +63,11 @@ class System < Katello::Model
 
     belongs_to :content_view, :inverse_of => :systems
 
+    has_one :capsule,
+            :class_name => "::SmartProxy",
+            :inverse_of => :system,
```

Comment:
this should be inverse_of  :content_host, as thats what you called it in smart_proxy_extensions